main.nf

```diff
@@ -613,32 +613,129 @@ process dia_spectral_library_search {
+        -Scoring:TransitionGroupPicker:PeakPickerMRM:signal_to_noise ${params.signal_to_noise} \\
+        -Scoring:TransitionGroupPicker:PeakPickerMRM:write_sn_log_messages ${params.write_sn_log_messages} \\
+        -Scoring:TransitionGroupPicker:PeakPickerMRM:remove_overlapping_peaks ${params.remove_overlapping_peaks} \\
+        -Scoring:TransitionGroupPicker:PeakPickerMRM:method ${params.method} \\
         -Scoring:TransitionGroupPicker:PeakIntegrator:integration_type 'intensity_sum' \\
         -Scoring:TransitionGroupPicker:PeakIntegrator:baseline_type 'base_to_base' \\
         -Scoring:TransitionGroupPicker:PeakIntegrator:fit_EMG 'false' \\
         -batchSize 1000 \\
         -readOptions ${params.cache_option} \\
         -tempDirectory tmp \\
+	-Scoring:DIAScoring:dia_extraction_window ${params.dia_extraction_window} \\
+        -Scoring:DIAScoring:dia_extraction_unit ${params.dia_extraction_unit} \\
+        -Scoring:DIAScoring:dia_centroided ${params.dia_centroided} \\
+        -Scoring:DIAScoring:dia_byseries_intensity_min ${params.byseries_intensity_min} \\
+        -Scoring:DIAScoring:dia_byseries_ppm_diff ${params.dia_byseries_ppm_diff} \\
+        -Scoring:DIAScoring:dia_nr_isotopes ${params.nr_isotopes} \\
+        -Scoring:DIAScoring:dia_nr_charges ${params.nr_charges} \\
+        -Scoring:DIAScoring:peak_before_mono_max_ppm_diff ${params.peak_before_mono_max_ppm_diff} \\
+        -Scoring:EMGScoring:max_iteration ${params.max_iteration} \\
+        -Scoring:Scores:use_shape_score ${params.use_shape_score} \\
+        -Scoring:Scores:use_coelution_score ${params.use_coelution_score} \\
+        -Scoring:Scores:use_rt_score ${params.use_rt_score} \\
+        -Scoring:Scores:use_library_score ${params.use_library_score} \\
+        -Scoring:Scores:use_intensity_score ${params.use_library_score} \\
+        -Scoring:Scores:use_nr_peaks_score ${params.use_nr_peaks_score} \\
+        -Scoring:Scores:use_total_xic_score ${params.use_total_xic_score} \\
+        -Scoring:Scores:use_total_mi_score ${params.use_total_mi_score} \\
+        -Scoring:Scores:use_sn_score ${params.use_sn_score} \\
+        -Scoring:Scores:use_mi_score ${params.use_mi_score} \\
+        -Scoring:Scores:use_dia_scores ${params.use_dia_scores} \\
+        -Scoring:Scores:use_ms1_correlation ${params.use_ms1_correlation} \\
+        -Scoring:Scores:use_sonar_scores ${params.use_sonar_scores} \\
+        -Scoring:Scores:use_ion_mobility_scores ${params.use_ion_mobility_scores} \\
+        -Scoring:Scores:use_ms1_fullscan ${params.use_ms1_fullscan} \\
+        -Scoring:Scores:use_ms1_mi ${params.use_ms1_mi} \\
+        -Scoring:Scores:use_uis_scores ${params.use_uis_scores} \\
         -Scoring:DIAScoring:dia_nr_isotopes 3 \\
         -enable_uis_scoring \\
         -Scoring:uis_threshold_sn -1 \\
         -threads ${task.cpus} \\
-        ${force_option} ${ms1_option} ${ms1_scoring} ${ms1_mi}
```

Comment:
The force_option parameter turned out to be very important for some analysis, I don't see why it needs to be removed.

In addition, I added boolean parameters in this way since I had problems running the OpenSwathWorkflow with setting them 'true' / 'false'. Correct me if I'm wrong.

Reply:
Readded.